Update create-your-first-custom-form.md

MattEEvans opened the pull request:
Updated Step 2 of "Create a custom form" to account for the new "Integrate" menu that replaced the "Power Apps" menu. I have an image too, but I don't know how to insert it. I tried to attach it here, but it doesn't seem to be working.

## Category

- [x] Content fix
- [ ] New article

## Related issues

I don't know of any issue number.

## What's in this Pull Request?

Updated Step 2 of "Create a custom form" to account for the new "Integrate" menu that replaced the "Power Apps" menu. I have an image too, but I don't know how to insert it. I tried to attach it here, but it doesn't seem to be working.

VesaJuvonen commented:
Docs Build status updates of commit _[ffcde06](https://github.com/MattEEvans/sp-dev-docs/commits/ffcde068a638cf26ed2b77b53de09944e36b7e4d)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) | [Details](#user-content-7c1a74bbf259fccd059103078d876e1e-7c7cbfc2)

<a id="7c1a74bbf259fccd059103078d876e1e-7c7cbfc2"></a>
### [docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md)
 - **Line 45, Column 156**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'Lists_Integrate_PowerApps_CreateApp' is duplicated. Within an article, alt text must be unique.`````
 - **Line 46, Column 65**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'Lists_Integrate_PowerApps_CreateApp' is duplicated. Within an article, alt text must be unique.`````
 - **Line 92, Column 5**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'View-only title' is duplicated. Within an article, alt text must be unique.`````
 - **Line 117, Column 5**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'View-only title' is duplicated. Within an article, alt text must be unique.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C6%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105062100187147-6992%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=7GIhL0XgcIEHda59M%2BhmHkr795CiLOiXVdCgCanXFM8%3D&st=2021-05-06T20%3A56%3A13Z&se=2021-06-06T21%3A01%3A13Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

andrewconnell commented on docs/business-apps/power-apps/get-started/create-your-first-custom-form.md:
```diff
@@ -42,7 +42,9 @@ Before starting, make sure your SharePoint site is set up with the [required lis
     1. When you select a control on the canvas, on the right-hand panel, the **Properties** pane associated with the control appears.
 
-    ![Image control with Properties panel](../../../images/gs01-image-control-selected.png)
+    ![Image control with Properties panel](../../../images/gs01-image-control-selected.png)<img width="403" alt="Lists_Integrate_PowerApps_CreateApp" src="https://user-images.githubusercontent.com/70417807/117352154-7bfb6380-ae63-11eb-9fc7-f275a62715f2.png">
```
Images should be added seperately as files to the folder structure, then reference them in the markdown. Can you please address before we can merge your submission? thanks!

MattEEvans replied:
I think I figured it out once I was looking in the MattEEvans:patch-1 branch. I submitted the update.

MattEEvans replied:
I discovered that my earlier attempt to upload a file as an attachment (the other day) had created some changes I didn't intend to make, so I backed those changes out. Hopefully we're good now.

andrewconnell replied:
@MattEEvans  please see the build errors/warnings in the last automated check. The image can't be found because it's in the wrong folder. It should be in the **./docs/images/** folder, while it appears you stuck it in the root **./images** folder. Can you please move the image in your PR to the correct location next to the other images in this article?

Also notice the build warning for the duped image `alt` tags. While you may not have been the author, we can't merge this with that error, so if you can update one of the alt tags to be more descriptive, we can process this further.

andrewconnell reviewed (changes requested):
Please see changes requested inline

VesaJuvonen commented:
Docs Build status updates of commit _[4c5db7f](https://github.com/MattEEvans/sp-dev-docs/commits/4c5db7f809f38748b2c9c7f2514fbdf013a7e173)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) | [Details](#user-content-7c1a74bbf259fccd059103078d876e1e-0c41c14f)
[images/lists-integrate-power-apps-create-app.png](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/images/lists-integrate-power-apps-create-app.png) | :white_check_mark:Succeeded | |

<a id="7c1a74bbf259fccd059103078d876e1e-0c41c14f"></a>
### [docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md)
 - **Line 45, Column 156**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'Lists_Integrate_PowerApps_CreateApp' is duplicated. Within an article, alt text must be unique.`````
 - **Line 46, Column 65**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'Lists_Integrate_PowerApps_CreateApp' is duplicated. Within an article, alt text must be unique.`````
 - **Line 92, Column 5**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'View-only title' is duplicated. Within an article, alt text must be unique.`````
 - **Line 117, Column 5**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'View-only title' is duplicated. Within an article, alt text must be unique.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C7%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105072202051688-6992%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=m5ncN6vGek%2BDe3eyPJ8t0wEwT6De3BDSvtahhGufozY%3D&st=2021-05-07T21%3A57%3A59Z&se=2021-06-07T22%3A02%3A59Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

VesaJuvonen commented:
Docs Build status updates of commit _[0dfefc8](https://github.com/MattEEvans/sp-dev-docs/commits/0dfefc82a31e8054c20fd3e0040c1d48207956d7)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) | [Details](#user-content-7c1a74bbf259fccd059103078d876e1e-1cb83fd7)
[images/lists-integrate-power-apps-create-app.png](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/images/lists-integrate-power-apps-create-app.png) | :white_check_mark:Succeeded | |

<a id="7c1a74bbf259fccd059103078d876e1e-1cb83fd7"></a>
### [docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md)
 - **Line 29, Column 5**: **[Warning-[file-not-found](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../../images/lists-integrate-power-apps-create-app.png'.`````
 - **Line 90, Column 5**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'View-only title' is duplicated. Within an article, alt text must be unique.`````
 - **Line 115, Column 5**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'View-only title' is duplicated. Within an article, alt text must be unique.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C7%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105072223051584-6992%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=M2oletOvbr0qilVhghdjTS2xQ6r6w0MItBw6JD4jgDo%3D&st=2021-05-07T22%3A19%3A05Z&se=2021-06-07T22%3A24%3A05Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

VesaJuvonen commented:
Docs Build status updates of commit _[79ea7e9](https://github.com/MattEEvans/sp-dev-docs/commits/79ea7e9b39d78d48f0345c97c66d004254e8f04d)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) | [Details](#user-content-7c1a74bbf259fccd059103078d876e1e-b6dabb02)

<a id="7c1a74bbf259fccd059103078d876e1e-b6dabb02"></a>
### [docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md)
 - **Line 29, Column 5**: **[Warning-[file-not-found](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../../images/lists-integrate-power-apps-create-app.png'.`````
 - **Line 90, Column 5**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'View-only title' is duplicated. Within an article, alt text must be unique.`````
 - **Line 115, Column 5**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'View-only title' is duplicated. Within an article, alt text must be unique.`````
 - **Line 37, Column 60**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/controls/control-image' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 37, Column 206**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/model-driven-apps/using-tree-view-on-form' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 47, Column 150**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/working-with-cards' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 49, Column 41**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/working-with-formulas' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 63, Column 192**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/connections/connection-office365-users' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 76, Column 19**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/controls/control-card' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 101, Column 24**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/controls/control-card' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 119, Column 83**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/functions/operators' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C17%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105172345155392-6992%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=T22P8EnRDo6ZRfGev8TVuvIJ%2Bfpastf3fMosBJAtDIM%3D&st=2021-05-17T23%3A41%3A17Z&se=2021-06-17T23%3A46%3A17Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

VesaJuvonen commented:
Docs Build status updates of commit _[86a8ad2](https://github.com/MattEEvans/sp-dev-docs/commits/86a8ad228d5a7e3db082e6ef3bb0b998501d6982)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) | [Details](#user-content-7c1a74bbf259fccd059103078d876e1e-1495b76a)
[docs/images/lists-integrate-power-apps-create-app.png](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/images/lists-integrate-power-apps-create-app.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) |

<a id="7c1a74bbf259fccd059103078d876e1e-1495b76a"></a>
### [docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md)
 - **Line 90, Column 5**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'View-only title' is duplicated. Within an article, alt text must be unique.`````
 - **Line 115, Column 5**: **[Warning-[duplicate-alt-text](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-alt-text?branch=main)]** `````Alt text 'View-only title' is duplicated. Within an article, alt text must be unique.`````
 - **Line 37, Column 60**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/controls/control-image' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 37, Column 206**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/model-driven-apps/using-tree-view-on-form' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 47, Column 150**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/working-with-cards' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 49, Column 41**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/working-with-formulas' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 63, Column 192**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/connections/connection-office365-users' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 76, Column 19**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/controls/control-card' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 101, Column 24**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/controls/control-card' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 119, Column 83**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/functions/operators' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C17%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105172354549719-6992%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=zeQkiJAvcFCL2zLCIPIwWKBhV%2BpEo7xiFTj2hXBkIwI%3D&st=2021-05-17T23%3A51%3A07Z&se=2021-06-17T23%3A56%3A07Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

VesaJuvonen commented:
Docs Build status updates of commit _[5f3b462](https://github.com/MattEEvans/sp-dev-docs/commits/5f3b462db591a67f3355335553cd45dd285fabdd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) | [Details](#user-content-7c1a74bbf259fccd059103078d876e1e-5406397e)
[docs/images/lists-integrate-power-apps-create-app.png](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/images/lists-integrate-power-apps-create-app.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) |

<a id="7c1a74bbf259fccd059103078d876e1e-5406397e"></a>
### [docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md)
 - **Line 37, Column 60**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/controls/control-image' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 37, Column 206**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/model-driven-apps/using-tree-view-on-form' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 47, Column 150**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/working-with-cards' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 49, Column 41**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/working-with-formulas' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 63, Column 192**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/connections/connection-office365-users' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 76, Column 19**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/controls/control-card' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 101, Column 24**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/controls/control-card' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 119, Column 83**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/powerapps/maker/canvas-apps/functions/operators' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C18%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105180006110470-6992%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=axfS9rVLbt%2Bk0%2BinRQaSuGJUZGctA7LRDHq8lzMaUDI%3D&st=2021-05-18T00%3A02%3A15Z&se=2021-06-18T00%3A07%3A15Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

MattEEvans commented:
OK, I've moved the image file to the **./docs/images/** folder, as requested. I also updated the duplicate alt text, which wasn't my doing. This isn't my article or my area of expertise, so I was hoping I wouldn't have to fix all the warnings. I'm new to editing in Docs and to this repo, so I didn't even realize I was putting the image in the wrong place. I now see a bunch of new "absolute link" errors in places where I didn't change anything. Do I need to change those? Geez, all I wanted was to make a small minor change. Didn't realize this would be such an arduous task!

andrewconnell commented:
Thanks for addressing the issue. The MSDocs team updates their content quality checker often so rules get added/updated after the original doc has been updated. There's a strong push for making docs more SEO friendly which is why you're seeing these.

Because these rules only run on docs when they are updated, such as the case with a PR, it finds "new" issues at times... like this one. We try to address each doc as it comes up while we're also working to take care of all the old stuff.

I'll handle the absolute/relative links issue... thanks for addressing the image & alt text.

VesaJuvonen commented:
Docs Build status updates of commit _[beaaced](https://github.com/MattEEvans/sp-dev-docs/commits/beaaced3a6b49fd50e2ef76e079bf95950bf7d2c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) |
[docs/images/lists-integrate-power-apps-create-app.png](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/images/lists-integrate-power-apps-create-app.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C18%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105181052455536-6992%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Yd0pd9WY7ZKuNFUIqA2kaiQoe9CbvxEdqPAvIRBa1b8%3D&st=2021-05-18T10%3A48%3A28Z&se=2021-06-18T10%3A53%3A28Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

VesaJuvonen commented:
Docs Build status updates of commit _[7bca05d](https://github.com/MattEEvans/sp-dev-docs/commits/7bca05d23b47aeb69678e4e46336c0dbfa590bea)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-apps/get-started/create-your-first-custom-form.md](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/business-apps/power-apps/get-started/create-your-first-custom-form.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) |
[docs/images/lists-integrate-power-apps-create-app.png](https://github.com/MattEEvans/sp-dev-docs/blob/patch-1/docs/images/lists-integrate-power-apps-create-app.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-apps/get-started/create-your-first-custom-form?branch=pr-en-us-6992) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C18%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105181052481272-6992%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=m2HycIStnEYIUz3OmfQ3B%2F%2BTkF69SoH5BqOq3oK1aUY%3D&st=2021-05-18T10%3A49%3A33Z&se=2021-06-18T10%3A54%3A33Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
